### PR TITLE
Add JWT support for danubetech.

### PR DESCRIPTION
**Pull request by peacekeeper**

### CODE REVIEW

File: implementations/danubetech/src/main/java/ldsignatures.test.jws/JWSTestSuite.java [hunk 1]
```diff
@@ -87,34 +43,83 @@ static void create(String input, String key, String outputFilename, String proof
...
+    }
+
+    static void createJwt(JwtObject jwtObject, URI verificationMethod, ByteSigner byteSigner, String outputFilename) throws JOSEException, IOException {
+        String jwtString;
+        if (JWSAlgorithm.EdDSA.equals(byteSigner.getAlgorithm()))
+            jwtString = jwtObject.sign_Ed25519_EdDSA(byteSigner, verificationMethod.toString(), false);
+        else if (JWSAlgorithm.ES256K.equals(byteSigner.getAlgorithm()))
+            jwtString = jwtObject.sign_secp256k1_ES256K(byteSigner, verificationMethod.toString(), false);
+        else if (JWSAlgorithm.PS256.equals(byteSigner.getAlgorithm()))
+            jwtString = jwtObject.sign_RSA_RS256(byteSigner, verificationMethod.toString(), false);
```
Comment (clehner) [inline implementations/danubetech/src/main/java/ldsignatures.test.jws/JWSTestSuite.java:64]: Should this be PS256?
```suggestion
            jwtString = jwtObject.sign_RSA_PS256(byteSigner, verificationMethod.toString(), false);
```

**Comment by clehner** [comment]
It built and ran on my system. The report shows Ed25519 and Secp256k1 VC-JWT and VP-JWT passing now. RSA does not appear.

**Comment by peacekeeper** [comment]
@clehner I agree with your fix, thanks.

However I don't think we properly support RSA at the moment, so I suggest we merge as-is for now and maybe I'll do another PR later.

**Comment by clehner** [comment]
Sounds good to me

**Review by clehner — APPROVED** [review]
Sounds good

File: implementations/danubetech/pom.xml [hunk 2]
```diff
@@ -54,9 +54,9 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>info.weboftrust</groupId>
-			<artifactId>ld-signatures-java</artifactId>
-			<version>0.5-SNAPSHOT</version>
+			<groupId>com.danubetech</groupId>
+			<artifactId>verifiable-credentials-java</artifactId>
```
Comment (OR13) [inline implementations/danubetech/pom.xml:58]: nice!

**Pull request merged**